app/backend/src/tests/test_admin.py

```diff
@@ -0,0 +1,167 @@
+import grpc
+import pytest
+
+from couchers import errors
+from couchers.db import session_scope
+from couchers.models import Cluster, User
+from couchers.servicers.admin import Admin
+from proto import admin_pb2, admin_pb2_grpc
+from tests.test_fixtures import generate_user, get_user_id_and_token, real_session, recreate_database
+
+
+@pytest.fixture(autouse=True)
+def _(testconfig):
+    pass
+
+
+NORMAL_USER_NAME = "normal_user"
+SUPER_USER_NAME = "super_user"
+NORMAL_USER_EMAIL = "normal@user.com"
+COMMUNITY_NAME = "test community"
+COMMUNITY_SLUG = "test-community"
+COMMUNITY_DESCRIPTION = "community for testing"
+VALID_GEOJSON_MULTIPOLYGON = """
+    {
+      "type": "MultiPolygon",
+      "coordinates":         
+       [ 
+        [
+          [
+            [
+              -73.98114904754641,
+              40.7470284264813
+            ],
+            [
+              -73.98314135177611,
+              40.73416844413217
+            ],
+            [
+              -74.00538969848634,
+              40.734314779027144
+            ],
+            [
+              -74.00479214294432,
+              40.75027851544338
+            ],
+            [
+              -73.98114904754641,
+              40.7470284264813
+            ]
+          ]
+        ]
+      ]
+    }
+"""
+INVALID_GEOJSON = """
+    {
+        "bla": "blah"
+    }
+"""
+
+
+def _session(token: str):
+    return real_session(token, admin_pb2_grpc.add_AdminServicer_to_server, Admin(), admin_pb2_grpc.AdminStub)
```

Comment:
`_session` is a bad name for this, I first thought it was a db session. Why not use `admin_session` or `real_admin_session`?